ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/queue_monitor_streaming.md

```diff
@@ -32,3 +35,24 @@ interface Management1
    vrf MGMT
    ip address 10.73.255.122/24
 ```
+
+## Queue Monitor
+
+### Queue Monitor Streaming
+
+
+| Enabled | IP Access Group | IPv6 Access Group | Max Connections | Vrf |
+| :-------: | :---------------: | :-----------------: | :---------------: | :---: |
+| True | ACL-QMS | ACLv6-QMS | 5 | test |
+
+### Queue Monitor Configuration
+
+```eos
+!
+queue-monitor streaming
+   max-connections 5
+   ip access-group ACL-QMS
+   ipv6 access-group ACL-QMS
```

Comment:
This is not rendering the correct ACL as compared to the table - is this a bug? If so we need a PR to fix it..

Reply:
Yeah it seems to me to be a bug. If you're ok with it, I'll add the fix in this PR as well.

Reply:
that's ok for me (will check your updates it may be done already)